game.js

```diff
@@ -44,7 +47,31 @@ function balloonKnot(x, y){
   ctx.fill();
 }
 
+function randomizer(num){
+  return Math.floor(Math.random() * num);
+}
+
+function colorPick(){ //yeah, we know having side effect to change global variable is not ideal
+  var pickColor = randomizer(3);
+  var result = pickColor===3?2:pickColor;
```

Comment:
es6 on frontend

Reply:
I don't think ternary is not es6.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
my bad ! sorry

Reply:
watching you mate ! watching you !